debian/ralf.init.d

```diff
@@ -156,6 +157,7 @@ do_start()
                      --log-file=$log_directory
                      --log-level=$log_level
                      $billing_realm
```

Comment:
This isn't your code, but whilst you're changing this file, do you fancy renaming billing_realm to billing_realm_arg to make it consistent? Be careful not to change the billing_realm config option that we pull in, just the locally defined variable!